doc/connectivity/networking/api/http_server.rst

```diff
@@ -29,6 +29,10 @@ Currently, the following resource types are supported:
 * Static resources - content defined compile-time, cannot be modified at runtime
   (:c:enumerator:`HTTP_RESOURCE_TYPE_STATIC`).
 
+* Static file system resources - file location defined compile-time, cannot be modified
+  at runtime (:c:enumerator:`HTTP_RESOURCE_TYPE_STATIC_FS`). The content of the files
+  might be immutable if the file system is read-only.
```

Comment:
does it make sense to (briefly) elaborate on what will happen if one tries to PUT/POST to these files? Or maybe I'm missing the point of what you're trying to explain when mentioning read-only file systems, since I'm not even sure "uploading" (to) files is supported?

Reply:
Only GET is supported for the file operations (at least for now). I can add more information to doc.